pages/Exotics.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react'
+import Page from '../components/Page'
+import Exotic from '../components/Exotic'
+import Search from '../components/Search'
+import NavLink from '../components/NavLink'
+import { retrieveExotics, filterExotics } from '../utils/exotics'
+
+export default () => {
+  const [searchTerm, setSearchTerm] = useState('')
+  const [exoticsList, setExoticsList] = useState([])
+  const [filteredExoticsList, setFilteredExoticsList] = useState([])
+
+  useEffect(() => {
+    async function pullData() {
+      const exotics = await retrieveExotics()
+
+      setExoticsList(exotics)
+      setFilteredExoticsList(exotics)
+    }
+
+    pullData()
+  }, [])
+
+  useEffect(() => {
+    const filtered = filterExotics(exoticsList, searchTerm)
+
+    setFilteredExoticsList(filtered)
+    localStorage.setItem('searchTerm', searchTerm)
+    console.log(searchTerm)
```

Comment:
This is why linters are helpful

![image](https://user-images.githubusercontent.com/5775716/85037529-7c0f4980-b153-11ea-8c03-d4a881f6c892.png)